### PR TITLE
Add a back to root collection button

### PR DESCRIPTION
**Pull request by cjmyers**

### CODE REVIEW

File: lib/getUrisFromReq.js [hunk 1]
```diff
@@ -23,6 +25,7 @@ function getUrisFromReq (req, res) {
       designId = designId + '/' + req.params.version
     }
     url = '/user/' + encodeURIComponent(req.params.userId) + '/' + designId
+    baseUrl = '/user/' + encodeURIComponent(req.params.userId) + '/' + req.params.collectionId
```
Comment (3ach) [inline lib/getUrisFromReq.js]: These strings are a bit confusing, they could be rewritten to have subparts clearly shown in larger parts:
```
let baseUrl = '/user/' + encodeURIComponent(req.params.userId) + '/' + collectionId
let url = baseUrl + '/' + req.params.displayId /* maybe + version */
let baseUri = config.get('databasePrefix') + baseUrl
```

etc. etc. Not really a requirement but could make the relationship between these parts easier to understand.

File: tests/previousresults/getrequest_public-:collectionId-:displayId(*)-:version-full_.html [hunk 2]
```diff
@@ -78,7 +78,7 @@ <h5>
         <h5>
          Upload New Icon
         </h5>
-        <form action="/public/testid0/testid0_collection/1/icon" class="form" enctype="multipart/form-data" id="iconForm" method="POST">
+        <form action="undefined/icon" class="form" enctype="multipart/form-data" id="iconForm" method="POST">
```
Comment (3ach) [inline tests/previousresults/getrequest_public-:collectionId-:displayId(*)-:version-full_.html]: Shouldn't be undefined.

File: tests/previousresults/getrequest_public-:collectionId-:displayId(*)-:version-full_.html [hunk 3]
```diff
@@ -320,39 +326,39 @@ <h4 class="panel-title">
            <tbody>
             <tr>
              <td>
-              synbiohub#topLevel
-              <a href="http://wiki.synbiohub.org/wiki/Terms/synbiohub#topLevel" title="Learn more about this annotation type">
+              synbiohub#ownedBy
```
Comment (3ach) [inline tests/previousresults/getrequest_public-:collectionId-:displayId(*)-:version-full_.html]: Is this a test reordering? We need to be careful checking in such changes (also need to fix them.)

**Comment by cjmyers** [comment]
Should be fixed now.  Also was not showing icon on collection page which is also fixed now.

**Pull request merged**